src/main/cpp/ffplay.cpp

```diff
@@ -1,16 +1,30 @@
 #include "FfmpegSdlAvPlayback.h"
```

Comment:
Could we add another method/project for this?

Reply:
I think that having an extra project for the MpvPlayer will be overkill, but I agree with having two methods to separate the two players.